src/protobuf/generator.cr

```diff
@@ -243,6 +243,11 @@ end
 
 module Protobuf
   class Generator
+    # Turns `foo/bar/batz.proto` into `foo_bar_batz.pb.cr`
+    def self.output_filename(path)
+      path.split(".")[0..-2].join("").gsub(File::SEPARATOR, "_") + ".pb.cr"
```

Comment:
Is there a reason not to nest the directory structure to match the `.proto` files?

Reply:
Not really, I just didn't want to think about creating all those sub-directories etc. 😅 

It seems most other proto generators indeed generate a nested structure
by default. One exception is the Golang one which makes a flat structure by
default unless you give a `paths=source_relative` parameter;
https://developers.google.com/protocol-buffers/docs/reference/go-generated

Don't really have a strong opinion about it myself (for me it's the difference
between `require "./protobuf/*"` vs `require "./protobuf/**"` 🤷 )